dbt_subprojects/daily_spellbook/models/_sector/utils/utils_quarters_table.sql

```diff
@@ -10,9 +10,12 @@
 SELECT timestamp
 FROM unnest(
     sequence(
-        timestamp '2009-01-01'
-        , cast(date_trunc('quarter', now() + interval '3' day) as timestamp)  -- add some padding to account for materialization lag
-        , interval '1' quarter
+          timestamp '2009-01-01'
+        , cast(
+            date_trunc('month', date_add('month', -mod(month(now() + interval '3' day) - 1, 3), now() + interval '3' day)) 
```

Comment:
unwarranted complexity?

Reply:
how would you do it instead? i just copied your logic from the existing utils tables

Reply:
what are you trying to do with `date_add('month', -mod(month(now() + interval '3' day) - 1, 3)` ?

Reply:
here is how it looks in query: https://dune.com/queries/5233264

Reply:
ah my bad, i know there's a function where `quarter` doesn't work in trinosql but i guess it's another one